package.json

```diff
@@ -80,5 +80,7 @@
     "webpack-cli": "^2.1.3",
     "webpack-dev-server": "^3.1.4"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@types/lodash": "^4.14.109"
```

Comment:
Why do we need this?

Reply:
I think we have lodash already so why not its type :D

Reply:
Nope, not in the package.json. Maybe some dependency uses it.

Do you need the typedefs? If not, then I don't really see a reason to keep it in 😉

Reply:
ok I will remove it